app/layout.tsx

```diff
@@ -53,11 +53,13 @@ const RootLayout = async ({children, modal}: {children: ReactNode; modal: ReactN
             Skip to main content
           </a>
         </nav>
-        {!previewMode && process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID && (
+        {!previewMode && process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID ? (
           <>
             <Script async src="//siteimproveanalytics.com/js/siteanalyze_80352.js" />
             <GoogleAnalytics gaId={process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID} />
           </>
+        ) : (
+          <GoogleAnalytics gaId="G-CEWCE1NE90" />
```

Comment:
@rebeccahongsf to revert before merging

Reply:
Yeah, maybe something like 
```
{!previewMode && (
  <>
    <Script async src="//siteimproveanalytics.com/js/siteanalyze_80352.js" />
    <GoogleAnalytics gaId={process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID || "G-CEWCE1NE90"} />
  </>
)}
``` 
would be cleaner here, but it hurts my heart to see hardcoded IDs even for testing.